src/collective/cover/static/css/cover.css

```diff
@@ -249,30 +249,23 @@ table.invisible
 }
 
 /* Tile Calendar */
-.cover-calendar-tile .calendar-tile-header
+.cover-calendar-tile .portletCalendar
```

Comment:
@hvelarde I have two options here.. or I do this way, or I have to customize 2 templates of both portlet.. you choose.